### PR TITLE
improve trading view trading mode

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
added:
ORDER_TYPE=MARKET       (/LIMIT)
PRICE=10000                      (optional)
VOLUME=0.004                  (optional)

### CODE REVIEW

File: Trading/Mode/trading_view_signals_trading_mode/trading_view_signals_trading.py [hunk 1]
```diff
@@ -104,9 +113,132 @@ def __init__(self, trading_mode):
...
+        self.SELL_WITH_MAXIMUM_SIZE_ORDERS = trading_mode.trading_config.get("use_maximum_size_orders", False)
         self.USE_STOP_ORDERS = False
 
+    async def create_new_orders(self, symbol, final_note, state, **kwargs):
+        data = kwargs.get("data", {})
+        user_price = float(data.get(TradingViewSignalsTradingMode.PRICE_KEY, 0))
+        user_volume = float(data.get(TradingViewSignalsTradingMode.VOLUME_KEY, 0))
+        current_order = None
+        timeout = kwargs.pop("timeout", trading_constants.ORDER_DATA_FETCHING_TIMEOUT)
+        try:
```
Comment (Herklos) [inline Trading/Mode/trading_view_signals_trading_mode/trading_view_signals_trading.py:126]: If from here it is a copy of the DailyTradingMode `create_new_orders`, we should probably consider making this code generic by making it available in a library.

File: Trading/Mode/daily_trading_mode/daily_trading.py [hunk 2]
```diff
@@ -244,6 +246,9 @@ def _check_and_adapt_order_details_if_necessary(self, quantity, price, symbol_ma
             decimal.Decimal(f"{quantity}"), decimal.Decimal(f"{price}"), symbol_market)
 
     async def create_new_orders(self, symbol, final_note, state, **kwargs):
+        data = kwargs.get("data", {})
```
Comment (Herklos) [inline Trading/Mode/daily_trading_mode/daily_trading.py:249]: I don't really like the name `data`, I would prefer something like `params` but it's ok

**Review by Herklos — APPROVED** [review]

File: Trading/Mode/daily_trading_mode/daily_trading.py (hunk 2 above)
Comment (GuillaumeDSM) [reply Trading/Mode/daily_trading_mode/daily_trading.py:249]: data is what we put there https://github.com/Drakkar-Software/OctoBot-Trading/blob/master/octobot_trading/modes/channel/abstract_mode_consumer.py#L39

Comment (Herklos) [reply Trading/Mode/daily_trading_mode/daily_trading.py:249]: Right :+1:

**Pull request merged**